components/color-picker/components/ColorPresets.tsx

```diff
@@ -59,9 +59,9 @@ const ColorPresets: FC<ColorPresetsProps> = ({ prefixCls, presets, value: color,
     children: (
       <div className={`${colorPresetsPrefixCls}-items`}>
         {Array.isArray(preset?.colors) && preset.colors?.length > 0 ? (
-          preset.colors.map((presetColor: Color) => (
+          preset.colors.map((presetColor: Color, index: number) => (
             <ColorBlock
-              key={`preset-${presetColor.toHexString()}`}
+              key={`preset-${index}-${presetColor.toHexString()}`}
```

Comment:
```suggestion
              key={`preset-${presetColor.toHexString()}-${index}`}
```

Reply:
Hi, I thought so, but in this case, the index refers to the preset, not the color. That is, preset-30 means the 30th color in the preset, and red-30 - it sounds like red 30 times repeated.
What do you think?

Reply:
You are right about it.